collector/ad.go

```diff
@@ -343,6 +344,12 @@ func newADCollector(logger log.Logger) (Collector, error) {
 			nil,
 			nil,
 		),
+		LdapClientSessions: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "client_sessions"),
+			"",
```

Comment:
I know this hasn't been done for the other `ad` metrics, but it'd be nice to have a description of the metric here. Does this metric show all current sessions, or the total number of sessions?